.github/workflows/test_iree.yml

```diff
@@ -67,12 +67,49 @@ jobs:
           source ${VENV_DIR}/bin/activate
           pytest iree_tests/onnx/node/generated -n auto -rpfE --timeout=30 --retries 2 --retry-delay 5 --durations=10
 
+  linux_x86_64_w7900_gpu_models:
+    strategy:
+      matrix:
+        os: [nodai-amdgpu-w7900-x86-64]
```

Comment:
If you keep the matrix, change "os" to something that matches the text and how it is used (maybe "runner"?)

Reply:
I was just thinking in the future, if we want to run the flow on different machines, but not needed now